web/apps/labelstudio/src/pages/CreateProject/Import/Import.scss

```
@@ -253,6 +252,13 @@
   margin-bottom: 24px;
 }
 
+.import-footer-ls {
```

```suggestion
.import-footer {
```
Won't BEM postfix `-ls` by default?

---

no BEM doesnt apply the suffix automatically. only the prefix